docs/releases/1.7.txt

```diff
@@ -637,7 +637,7 @@ Forms
   value.
 
 * :attr:`SelectDateWidget.months
-  <django.forms.extras.widgets.SelectDateWidget.months>` can be used to
+  <django.forms.SelectDateWidget.months>` can be used to
```

Comment:
IMO, these changes (also in docs/releases/1.8.txt) are not necessary since this PR will only go in master.